Nos/Models/PublicKey.swift

```diff
@@ -21,7 +21,11 @@ struct PublicKey {
     let bytes: [UInt8]
      
     private let underlyingKey: secp256k1.Signing.XonlyKey
-    
+
+    static func build(_ npubOrHex: String) -> PublicKey? {
```

Comment:
Could we instead do this?

```
init?(npubOrHex: String) {
    init(npub: npubOrHex) ?? init(hex: npubOrHex)
}
```

Reply:
I tried to do something similar but it didn't compile. Let me try that exact code you say.

Reply:
No, it doesn't work either.